### DISCUSSION TITLE
Edited Measure Not Visibly Distinct from Non-Edited Measures

### DISCUSSION THREAD

**Discussion by MuddasarM** [TabularEditor/TabularEditor3#1046] [Ideas]
In TE2 an asterisk would appear next to measure names (or maybe the name went bold) once their changes were accepted and before changes had been saved to the database.

In TE3 this feature doesn't appear to be present.

**Comment by otykier** [comment]
TE2 uses this feature to indicate that the object could potentially have a syntax error (since TE2 doesn't validate the DAX code). TE3, however, is perfectly able to validate the code without saving to the database.

TE3 will display a warning bar if you open a DAX query view or a Pivot Grid while there are local changes, that have not yet been sync'ed to Analysis Services. I'd like to understand what the use case of having this information on the individual objects are, so may I ask you to describe in more detail what the benefit of that would be? Thanks!

**Comment by MuddasarM** [comment]
When I am editing multiple measures in a large model, sometimes I can accidentally paste DAX into the wrong measure. The highlighted measure, in TOM Explorer, indicating local changes that have not been saved can then save me from unintentionally saving a change I didn't intend to make. 

In TE2 it has saved me many times. In TE3 I'm suddenly having to pay more attention; just feel it should be making my life easier. It's very useful feedback from the tool that these measures have local changes that have yet to be saved.

**Comment by otykier** [comment]
I see. Thanks for clarifying. Will definitely consider it for a future update.

**Comment by mlonsk** [comment]
Hi @MuddasarM 
We have in Tabular Editor 3.27.0.0 published [unsaved change indicators](https://docs.tabulareditor.com/en/features/unsaved-changes.html) which in our opinion gives an even more comprehensive experience that TE2s validation - happy to say that we can finally close this feature request
